File: app/src/main/java/org/pocketx/knell/domain/BirthdayManager.java
package org.pocketx.knell.domain;

/**
 * @author wd
 */
public interface BirthdayManager {

    void set(Birthday birthday);

    Birthday get();

    boolean exist();

    /**
     * 删除生日数据
     */
    void deleteBirthday();
}

File: app/src/main/java/org/pocketx/knell/domain/BirthdayManager.kt
package org.pocketx.knell.domain

/**
 * @author wd
 */
interface BirthdayManager {

    fun set(birthday: Birthday)

    fun get(): Birthday

    fun exist(): Boolean

    /**
     * 删除生日数据
     */
    fun deleteBirthday()
}
